src/Test/L0/Listener/Configuration/ConfigurationManagerL0.cs

```diff
@@ -234,5 +234,68 @@ public async Task ConfigureErrorOnMissingRunnerGroup()
                 _runnerServer.Verify(x => x.GetAgentPoolsAsync(It.IsAny<string>(), It.Is<TaskAgentPoolType>(p => p == TaskAgentPoolType.Automation)), Times.Exactly(1));
             }
         }
+
+#if OS_LINUX
```

Comment:
```suggestion
#if !OS_WINDOWS
```

Reply:
Won't this include MacOS? I didn't think we wanted that.

Reply:
👍